src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star active_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+        <div class="stars__star deactive_star"></div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__sta"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__starr"></div>
      </div>
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
additional classes are redundant.
stars--1, stars--2 ... modifiers should be responsible of amount active stars.